src/content/configuration/optimization.md

```diff
@@ -324,4 +324,23 @@ module.exports = {
     concatenateModules: true
   }
 };
-```
\ No newline at end of file
+```
+
+## `optimization.sideEffects`
+
+`bool`
+
+Tells webpack to recognise the sideEffects flag in `package.json` or rules to skip over modules which are flagged to contain no side effects when exports are not used. `optimization.sideEffects` depends on [`optimization.providedExports`](#optimization-providedexports) and [`optimization.usedExports`](#optimization-usedexports). These dependencies have a build time cost, but eliminating modules has positive impact on performance because of less code generation. Effect of this optimization depends on your codebase, try it for possible performance wins.
```

Comment:
> flags or rules in `package.json`

Reply:
The rules maybe relate to the `module` rules?

Reply:
CC @ooflorent